### PR TITLE
First Solution of cinema

### PR DESCRIPTION
**Pull request by svyshcho**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str,
+                      customers: list[Customer],
+                      cleaning_staff: Cleaner) -> None:
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        for cust in customers:
```
Comment (ArsenPidhoretskyi) [inline app/cinema/hall.py:13]: Rename cust to customer

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    new_bar = CinemaBar
```
Comment (ArsenPidhoretskyi) [inline app/main.py:11]: You must create instance
bar = CinemaBar()

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+    for customer in customers:
+        new_customer = Customer(customer.get("name"), customer.get("food"))
```
Comment (ArsenPidhoretskyi) [inline app/main.py:13]: rename new_customer to customer_obj

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+        new_bar.sell_product(new_customer.food, new_customer)
+    new_session = CinemaHall(hall_number)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:15]: new_session -> cinema_hall

File: app/main.py [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
+    list_of_customers = [Customer(name=customer["name"],
```
Comment (ArsenPidhoretskyi) [inline app/main.py:16]: list_of_customers -> customers_objs

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Check naming)

File: app/main.py [hunk 6]
```diff
@@ -1,6 +1,20 @@
...
+                 cleaner: str,
+                 movie: str) -> None:
+    cinema_bar = CinemaBar()
+    for customer in customers:
+        customer_obj = Customer(customer.get("name"), customer.get("food"))
+        cinema_bar.sell_product(customer_obj.food, customer_obj)
+    cinema_hall = CinemaHall(hall_number)
+    customers_objs = [Customer(name=customer["name"],
+                               food=customer["food"])
+                      for customer in customers]
```
Comment (olena-hul) [inline app/main.py:18]: You're duplicating the creation of Customer instances. It can be done in a loop above

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 6 above)
Comment (svyshcho) [reply app/main.py]: Done

File: app/main.py [hunk 7]
```diff
@@ -1,6 +1,20 @@
...
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    cinema_bar = CinemaBar()
+    customers_objs = []
+    for customer in customers:
+        customer_obj = Customer(customer.get("name"), customer.get("food"))
+        cinema_bar.sell_product(customer_obj.food, customer_obj)
+        customers_objs.append(Customer(name=customer["name"],
+                                       food=customer["food"]))
```
Comment (olena-hul) [inline app/main.py:17]:
```suggestion
        customers_objs.append(customer_obj)
```

You have already initialized customer_obj)

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 7 above)
Comment (svyshcho) [reply app/main.py]: Didn't notice))
Done

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**